docs/deploy-on-k8s.md

```diff
@@ -0,0 +1,115 @@
+# Deploying on Kubernetes
+
+In order to deploy the application to Kubernetes we have provided some manifests and helper scripts in the `./deployments` directory.
+
+The scripts will help you:
+- Create a Kubernetes cluster on EKS, if you don't already have one you'd like to use
+- Deploy a basic Temporal cluster to the Kubernetes cluster, if you don't already have a Temporal install you'd like to use
+- Deploy the OMS application into the Kubernetes cluster
+- Access the OMS Web UI and Temporal Web UI from your local machine
+
+## Creating a Kubernetes cluster
+
+We have provided a script to create a new EKS cluster on AWS, but using EKS is not a requirement. If you already have a Kubernetes cluster you would like to use, from AWS or any other provider, please feel free to skip this step. Any Kubernetes cluster will work.
+
+In order to use the script please ensure you have valid credentials setup to use the AWS APIs. Setting up authentication is beyond the scope of this document, but you can find some details on what IAM policies you may need in the [`eksctl` documentation](https://eksctl.io/usage/minimum-iam-policies/).
+
+To create the cluster (called "temporal-oms"):
+
+```sh
+./deployments/create-eks.sh
+```
+
+This will create a new EKS cluster on AWS and configure `kubectl` to talk to it.
+
+You should be able to run:
+
+```sh
+kubectl get pods
+```
+
+And see the message `No resources found in default namespace.`
+
+## Installing Temporal into Kubernetes
+
+To install a basic Temporal cluster into Kubernetes we can use the Temporal Helm Charts.
+
+To install Temporal:
+
+```sh
+./deployments/install-temporal.sh
+```
+
+This may take a while, as the Temporal cluster relies on Cassandra and Elasticsearch, which can both take a while to boot up.
+
+Once the Temporal cluster is up, the script creates the Temporal namespace "default" for you. This is Temporal namespace the OMS application expects to use, unless configured otherwise.
```

Comment:
```suggestion
Once the Temporal cluster is up, the script creates the Temporal namespace `default` for you. This is the Temporal namespace the OMS application expects to use, unless configured otherwise.
```